tests/e2e-playwright/tests/jupyterlab.py

```diff
@@ -69,6 +69,10 @@ def test_jupyterlab(
         page.frame_locator(".qx-main-dark").locator(
             ".jp-LauncherCard-icon"
         ).first.click()
+        page.wait_for_timeout(3000)
```

Comment:
careful with hardcoded timeouts!  At some point you will lose control of how many you have around ... 
I would create a rationale about these and put them in a central place so you can tune the whole setup properly

Reply:
Agree, unfortunately, this is on a case-to-case basis and sometimes timeouts are needed in e2e tests.